cmd/marketplace/server.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/mattermost/mattermost-marketplace/internal/api"
+	"github.com/mattermost/mattermost-marketplace/internal/model"
+	"github.com/mattermost/mattermost-marketplace/internal/store"
+	"github.com/pkg/errors"
+	logrus "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var instanceID string
+
+func init() {
+	instanceID = model.NewID()
+
+	serverCmd.PersistentFlags().String("database", "plugins.json", "The read-only JSON file backing the server.")
+	serverCmd.PersistentFlags().String("listen", ":8085", "The interface and port on which to listen.")
```

Comment:
Just wondering if it's worth having these settings in a config file from the start?

Reply:
I'm inclined to adopt the [12-factor](https://12factor.net/config) school of thought here and prefer the configuration to be defined in the environment vs. a config file. Of course, missing is the ability to actually employ the environment here, but that's basically just a `viper` call away in a subsequent PR. Hoping we can avoid the configuration file altogether, at least in terms of what's checked into the repository.